db/versions/0047.yml

```diff
@@ -0,0 +1,39 @@
+version: 47
+description: add get_non_stopped_workers db function
```

Comment:
Up to you as it's a matter of preference, but `get_active_workers` might be a less awkward name..

Reply:
I actually like `non_stopped` I think. `active` feels like maybe or maybe not it will include `stopping`?